components/VotesList/VotesList.tsx

```diff
@@ -0,0 +1,37 @@
+import { tabletAndUnder, tabletMax } from "constant";
```

Comment:
This is the same `VotesTable` component as before. I renamed it because it is not always a table — on mobile it is a list of cards. The behavior is the same on desktop and mobile other than using different element types.